layout/_partials/footer.swig

```diff
@@ -1,6 +1,9 @@
 <div class="copyright">{#
 #}{% set current = date(Date.now(), "YYYY") %}{#
-#}&copy; {% if theme.footer.since and theme.footer.since != current %}{{ theme.footer.since }} – {% endif %}{#
+#}{% if theme.footer.beian.enable %}{#
+#}<span class="with-love">{{theme.footer.beian.icp}}</span>{#
```

Comment:
Also, there is no need to add `span class="with-love`. Please, add only service here.